### PR TITLE
Updated the doc

### PR DESCRIPTION
**Pull request by joinimran**
Under requirements, Windows 10 pro or enterprise version 1703 doesn't need to be activated before applying Win10E3 license.

### CODE REVIEW

File: windows/deployment/windows-10-enterprise-subscription-activation.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ The following figure illustrates how deploying Windows 10 has evolved with each
 
 For Microsoft customers with Enterprise Agreements (EA) or Microsoft Products & Services Agreements (MPSA), you must have the following: 
 
-- Windows 10 (Pro or Enterprise) version 1703 or later installed and **activated** on the devices to be upgraded.
+- Windows 10 (Pro or Enterprise) version 1703 or later installed on the devices to be upgraded.
```
Comment (kenwith) [inline windows/deployment/windows-10-enterprise-subscription-activation.md]:
```suggestion
- Windows 10 (Pro or Enterprise) version 1703 or later installed on the devices to be upgraded. 
```

File: windows/deployment/windows-10-enterprise-subscription-activation.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ The following figure illustrates how deploying Windows 10 has evolved with each
...
```
Comment (kenwith) [inline windows/deployment/windows-10-enterprise-subscription-activation.md]:
```suggestion
- Windows 10 (Pro or Enterprise) version 1703 or later installed on the devices to be upgraded.
```

**Comment by mypil** [comment]
@greg-lindsay - can you please assist to check, approve and merge the content for this PR based on issue #3074?

Thank you.

**Comment by mypil** [comment]
@greg-lindsay - I would like to follow up. can you please assist to check, approve and merge the content for this PR based on issue #3074?

Thank you.

**Comment by mypil** [comment]
@greg-lindsay - I would like to follow up. can you please assist to check, approve and merge the content for this PR based on issue #3074?

Thank you.

**Pull request merged**